README.md

```diff
@@ -36,6 +36,15 @@ To run your bot against a live network, this SDK includes a simple runner you ca
 $ silverback run "example:app" --network :mainnet:alchemy
 ```
 
+## Docker Usage
+
+```sh
+$ docker build -t silverback .
+$ docker run --volume $PWD:/home/harambe/project \
+silverback run "example:app" \
```

Comment:
When doing multi-line BASH statements, might want to indent on the lines after the first so it is more readable

Reply:
should I indent these or just put them on one line?

Reply:
fixed